### PR TITLE
Update Elixir, Erlang and Protobuf

### PR DESCRIPTION
**Pull request by britto**
* Update the CI matrix
* Bump Elixir and Erlang
* ~Drop Elixir 1.7 and Erlang 20.0 support~
* Bump Protobuf and recompile internal protos

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -7,7 +7,7 @@ defmodule Protobuf.Mixfile do
     [
       app: :protobuf,
       version: @version,
-      elixir: "~> 1.6",
```
Comment (whatyouhide) [inline mix.exs:10]: If possible, let's not bump the version yet. I don't see a strong enough reason yet :)

Comment (britto) [reply mix.exs:10]: Good point. I've reverted the changes in #109 that required 1.8+ too.

Comment (britto) [reply mix.exs]: Done.

**Comment by whatyouhide** [comment]
Awesome!

**Pull request merged**